Sources/FoundationExtensions/UIApplication+RCExtensions.swift

```diff
@@ -18,6 +18,7 @@ import UIKit
 extension UIApplication {
 
     @available(iOS 13.0, *)
+    @available(macCatalystApplicationExtension 13.1, *)
```

Comment:
Hmm should this also be available in `macCatalyst` (not an application extension)?

Reply:
`iOS` already satisfies that requirement implicitly (`iOS` is basically `macCatayst` too unless you add a specific `@available` for it).
Which means that this is available for `macCatalyst 13.0`. I have no idea why Apple changed the requirement here in Xcode 15, but either way this property is only used from places that require iOS 15, which is why everything else comiles.